src/hooks/posters.tsx

```diff
@@ -162,5 +182,31 @@ export const usePosters = (posterHallId: string) => {
     setSearchInputValue,
     setLiveFilter,
     setBookmarkedFilter,
-  };
+  } as PostersContextState;
```

Comment:
Using `as` isn't type safe, and shouldn't be needed.

Reply:
done

Reply:
Please add the commit hash of changes made to comment threads when resolving as per our standards